### ISSUE TITLE
prompt-toolkit error between objection and frida

### ISSUE THREAD

**Issue by xorcat** [sensepost/objection#321]
Similar to [this issue](https://github.com/sensepost/objection/issues/179), `frida` has [just updated](https://github.com/frida/frida-tools/commit/25f614d71186a4aaa8a6b360fc865e364adaeca7#diff-2eeaed663bd0d25b7e608891384b7298) their dependency on `prompt-toolkit` to 3.x.

I'm now getting the following error when trying to launch `objection`, which I think I'll be able to work around this by pinning to an older version of `frida-tools` (to be confirmed).

However, it looks like it's time to update to `prompt-toolkit` 3.x :smiley: 

Thank you!

**Comment by xorcat** [comment]
```
pkg_resources.ContextualVersionConflict: (prompt-toolkit 2.0.10 (/home/user/.local/lib/python3.8/site-packages), Requirement.parse('prompt-toolkit<4.0.0,>=3.0.3'), {'frida-tools'})
```

**Comment by xorcat** [comment]
Update: Updating requirements.txt to the following (note `frida-tools`) got things working for me until objection is updated.

```
frida
frida-tools==6.0.0
prompt_toolkit>=2.0.9,<3.0.0
click
tabulate
delegator.py
requests
flask
pygments
litecli==1.1.0
```

**Comment by leonjza** [comment]
Thanks for reporting this! I'll fix it up later today. Looks like a _really_ simple upgrade to 3 when comparing to 2. https://python-prompt-toolkit.readthedocs.io/en/master/pages/upgrading/3.0.html

**Comment by leonjza** [comment]
I have temporarily pinned the `frida-tools` version until https://github.com/dbcli/litecli/issues/83 is resolved.

**Comment by leonjza** [comment]
For now, version 1.8.4 should install fine.

**Comment by leonjza** [comment]
I just did some maintenance on all of the dependencies objection uses (both python & JavaScript) which should land in the next release fixing this as well. Commits: https://github.com/sensepost/objection/commit/326b28ec330802cfea0509d96e0c7a1551125c37 https://github.com/sensepost/objection/commit/e09e7bda54e188b3119b0c3c52e6be9f4cb68860

**Comment by blshkv** [comment]
litecli-1.3.0 is out, could you push fixed version?

**Comment by leonjza** [comment]
@blshkv  As soon as the next release is ready, yes.

**Comment by blshkv** [comment]
FYI, frida-tools 6.0.0 is the last version with ```"prompt-toolkit >= 2.0.0, < 3.0.0"``` so objection 1.8.4 should allow this version.

**Comment by leonjza** [comment]
Should be fixed in 1.9.0.
